README.md

```diff
@@ -0,0 +1,177 @@
+For Node.js Developers
+======================
+
+The JSON Schema Test Suite is meant to be a language agnostic test suite for testing JSON Schema validation libraries.
+It is generally added to projects as a git submodule. However, to simplify things for Node.js developers, the test suite has also
+been made available as an npm package.
+
+    npm install json-schema-test-suite
+
+### Usage:
+
+There are a number of ways to load tests from the suite:
+
+    const testSuite = require('json-schema-test-suite');
+
+    // this will load all (required and optional) draft6 tests
+    const tests = testSuite.loadSync();
+
+    // optional `filter` is a function that takes 3 arguments (filename, parent, optional)
+    // and returns true if the test should be included. The optional argument is true
+    // for all files under the `<draft>/optional` directory.
+    // optional `draft` should be either `'draft3'`, `'draft4'` or `'draft6'`
+
+    const tests = testSuite.loadSync(filter, draft);
+
+    // convenience functions:
+
+    // The following take an optional `filter` as described previously (undefined will load all tests)
+    const draft3 = testSuite.draft3();
+    const draft4 = testSuite.draft4();
+    const draft6 = testSuite.draft6();
+
+    // The following take an optional `draft` argument (defaults to 'draft6')
+    const all = testSuite.loadAllSync();
+    const required = testSuite.loadRequiredSync();
+    const optional = testSuite.loadOptionalSync();
+
+
+The return value of these functions is an array of objects that correspond to each file under `tests/<draft>` that
+passed the filter (the default is all, so the array will also include all the optional files).
+
+Each object has the following structure (using `tests/draft4/additionalItems.json` as an example):
+
+```
+{
+  name:    'additionalItems',
+  file:     'additionalItems.json',
+  optional: false,  // true if a file under the optional directory
+  path:     '/full/path/to/JSON-Schema-Test-Suite/tests/draft4/additionalItems.json',
```

Comment:
Just out of curiosity, is the `path` actually useful in practice? Same about the `file` one. Is there a case where you want to know that? If not, maybe worth omitting them, and only add them if someone has a good use case for it

Reply:
Yep I agree, its not that useful